library/tasks/StarMkref.wdl

```diff
@@ -7,7 +7,10 @@ task StarMkref {
   String star_docker_image = "quay.io/humancellatlas/secondary-analysis-star:v0.2.2-2.5.3a-40ead6e"
```

Comment:
if you want to put docker name as variable, you'd better pass them in the input file, right?  we can put all docker names in a json files, such as docker.json and pass it to wdl, is it possible?

Reply:
Heh, I forgot I'd done this. Yea... we could change this across our wdls.